pkg/config/servicebuilder.go

```diff
@@ -103,6 +104,12 @@ func (bldr *ServiceBuilder) WithStorageService() *ServiceBuilder {
 	return bldr
 }
 
+// WithDataService tells the builder to add the Data service to the `ConfigurationBuilder`
+func (bldr *ServiceBuilder) WithDataService() *ServiceBuilder {
```

Comment:
Since this appears to be just creating the data service for account, is this name too generic?